site/content/reference/openshift-permissions.md

```diff
@@ -0,0 +1,44 @@
+| -------------------------- | ------------------------------------------------------ | ---------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
+| _allowPrivilegedContainer_ | _false_                                                | None of the containers in the NGINX Gateway Fabric Pod need to be run as privileged.                                                                                                                 |
+| _allowHostDirVolumePlugin_ | _false_                                                | None of the containers in the NGINX Gateway Fabric Pod need to use the HostDir volume plugin.                                                                                                        |
+| _allowHostIPC_             | _false_                                                | None of the containers in the NGINX Gateway Fabric Pod need host ipc.                                                                                                                                |
+| _allowHostNetwork_         | _false_                                                | The NGINX Gateway Fabric Pod does not require the use of HostNetwork.                                                                                                                                |
+| _allowHostPorts_           | _false_                                                | None of the containers in the NGINX Gateway Fabric Pod need to use host ports.                                                                                                                       |
+| _readOnlyRootFilesystem_   | _true_                                                 | None of the containers in the NGINX Gateway Fabric Pod require a non-read only root file system.                                                                                                     |
+| _runAsUser_                | _MustRunAsRange: 101-102_                              | The NGINX Gateway Fabric Pod sets the user ID for the nginx-gateway and nginx containers to 102 and 101 respectively.                                                                                |
+| _fsGroup_                  | _MusRunAs: 1001-1001_                                  | The NGINX Gateway Fabric Pod sets the nginx container's fsGroup to 1001.                                                                                                                             |
+| _supplementalGroups_       | _MustRunAs: 1001-1001_                                 | Since the nginx container's fsGroup is set to 1001, all processes of the container are also part of the supplementary group ID 1001.                                                                 |
```

Comment:
I'm not following this explanation. What are supplemental groups? Is that an openshift specific thing? Does the supplemental group ID always need to match the filesystem group ID? What error do you get if you omit this value?